h/static/styles/partials-v2/_api-token.scss

```diff
@@ -0,0 +1,14 @@
+.api-token {
+  border: 1px solid $grey-3;
+  border-radius: 3px;
+  display: block;
+  min-height: 50px;
+  width: 100%;
+
+  color: $grey-6;
+  text-align: center;
```

Comment:
The font size looks larger than the default in the mock at https://trello-attachments.s3.amazonaws.com/577a6fb9667c37f497050532/1440x800/e84807532e003d6a2d2ed386af011405/settings_-_developer_regenerate.png . I think this would be `$input-font-size`